pkgs/tools/security/hashcash/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation rec {
+  name = "hashcash-${version}";
+  version = "1.22";
+
+  buildInputs = [ openssl ];
+
+  src = fetchurl {
+      url = "http://www.hashcash.org/source/hashcash-1.22.tgz";
+      sha256 = "15kqaimwb2y8wvzpn73021bvay9mz1gqqfc40gk4hj6f84nz34h1";
+  };
+
+  makeFlags = "generic-openssl LIBCRYPTO=-lcrypto";
+
+  installPhase = ''
```

Comment:
Try the following instead of manually writing `instalPhase`:

```nix
installFlags = [
  "INSTALL_PATH=${placeholder "out"}/bin"
  "MAN_INSTALL_PATH=${placeholder "out"}/share/man/man1"
  "DOC_INSTALL_PATH=${placeholder "out"}/share/doc/hashcash-$(HC_VERS)"
];
```

Reply:
This is better.  I made this change in 6b8c09bffca82c1373c38586cac2032088f0e5bf